src/main.rs

```diff
@@ -6,28 +6,28 @@ use std::str::FromStr;
 
 fn main() -> Result<(), Box<dyn Error>> {
     let matches = App::new("fastqc-rs")
-        .version("0.1")
+        .version("0.3.1-alpha.1")
```

Comment:
Not hardcoding this would have probably been a much better idea from the first place:

```suggestion
        .version(env!("CARGO_PKG_VERSION"))
```